requirements/requirements.in

```diff
@@ -1,20 +1,19 @@
-aiohttp>=3.7.4
+aiohttp
 ansiconv==1.0.0  # UPGRADE BLOCKER: from 2013, consider replacing instead of upgrading
 asciichartpy
 asn1
-autobahn>=20.12.3  # CVE-2020-35678
 azure-keyvault==1.1.0  # see UPGRADE BLOCKERs
 channels
-channels-redis>=3.1.0  # https://github.com/django/channels_redis/issues/212
-cryptography>=36.0.2,<37.0.0 # Until paramiko fixes https://github.com/paramiko/paramiko/issues/2038 we don't want to go to 37 or we end up with blowfish warnings in the job output
+channels-redis
+cryptography
```

Comment:
were the blowfish warnings resolved?

Reply:
We did manual testing with SSH keys and `connection: paramiko` and did not find any blowfish warnings.